pom.xml

```diff
@@ -324,4 +324,59 @@
 		<module>spring-cloud-netflix-sidecar</module>
 		<module>docs</module>
 	</modules>
+	
+	<!-- 
+		Profile used by the Eclipse Maven plugin (m2eclipse).
+	
+		Activation:
+			The M2E Eclipse plugin exports the property "m2e.version" automatically when installed.
+			This profile is automatically activated when that property is found, meaning the build
+			is running under Eclipse...
+		
+		Additional doc:
+			https://docs.sonatype.org/display/M2ECLIPSE/Customizable+build+lifecycle+mapping+for+m2e+extensions+developers
+			http://wiki.eclipse.org/M2E
+	 -->
+	<profiles>
```

Comment:
Can you do a separate pull request for this?  I'm not sure this project is the right place anyway.  [spring-cloud-build](https://github.com/spring-cloud/spring-cloud-build) might be more appropriate.